seqr/views/utils/test_utils.py

```diff
@@ -252,7 +252,7 @@ def get_initial_page_json(self, response):
         'test_user_manager@test.com': {
             "accessLevel": "WRITER",
             "pending": False,
-            "canShare": True,
+            "canShare": False,
```

Comment:
It is changed for testing a manager user who doesn't have workspace permission for adding workspace data.

Reply:
wouldn't it make more sense for the manager user to be able to share and the collaborator user to not be able to share? If that makes the tests way more complicated no need to change it, but if it can be done either way then swapping it makes sense

Reply:
Yes. It makes more sense but is not necessary. Before, I was trying to create a scenario where a user passes the local permission check but doesn't have enough permission on Anvil. So I had to change the manager user. Since we have changed the permission check, I can use the collaborator user rather than the manager user to do the test.